pyiron_base/_tests.py

```diff
@@ -88,4 +89,52 @@ def tearDown(self):
         self.project.remove_jobs_silently(recursive=True)
 
 
-_TO_SKIP = [PyironTestCase, TestWithProject, TestWithCleanProject]
+class ToyJob(PythonTemplateJob):
+    def __init__(self, project, job_name):
+        """A toyjob to test export/import functionalities."""
+        super(ToyJob, self).__init__(project, job_name)
+        self.input['input_energy'] = 100
+
+    # This function is executed
+    def run_static(self):
+        with self.project_hdf5.open("output/generic") as h5out:
+            h5out["energy_tot"] = self.input["input_energy"]
```

Comment:
```suggestion
        self.status.running = True
        self.output.data_out = self.input.data_in + 1
```

The `running` is just trivial to set a good example. The benefit of leaning on the `DataContainer` interface is clear though, since we never need to talk about hdf.

Aha, no, wait. I still need to add a final `to_hdf()`. In the future hopefully this will be handled by the `.run()` flow too, but for now we'll live with it.